themes/addons/templates/Layout/Addons.ss

```diff
@@ -1,5 +1,5 @@
 <div class="page-header">
-	<h1>Modules and Themes <a class="pull-right" href="rss"><img src="themes/addons/images/feed-icon-28x28.png" width="20" alt="RSS Feed" /></a></h1>
+	<h1>Modules and Themes <a class="pull-right" href="/add-ons/rss"><img src="themes/addons/images/feed-icon-28x28.png" width="20" alt="RSS Feed" /></a></h1>

```

Comment:
this did not seem to fix the problem locally, it's because the Link method is explicitly set in the AddonsController.

Possible we could do "{$Link}/rss" ?